### PR TITLE
[vcpkg baseline][tinyfiledialogs] Change repo to sourceforge

### PR DESCRIPTION
**Pull request by JackBoosY**
Since the repo `https://git.code.sf.net` is unavaliable (404), change it to sourceforge.

Note the `current` version is `3.8.8`.

### CODE REVIEW

**Comment by JackBoosY** [comment]
cc @SamuelMarks

**Review by PhoebeHui — APPROVED** [review]

**Comment by SamuelMarks** [comment]
@JackBoosY Hmm that doesn't work @ f596542f6:
```sh
$ gh pr checkout 19287
$ ./bootstrap-vcpkg.sh
$ ./vcpkg install tinyfiledialogs
```

Then in another dir with this above it:
```cmake
cmake_minimum_required(VERSION 3.4)
cmake_policy(SET CMP0048 NEW)
project(pp VERSION 0.0.0 LANGUAGES C)

file(DOWNLOAD "https://sourceforge.net/p/tinyfiledialogs/code/ci/4b65621952f2cf1045837a6380aa912b028daa47/tree/hello.c?format=raw"
              "src/main.c"
     EXPECTED_HASH SHA256=72502b4b5e93bf262b1ff148de028d12a187c1fda45de89212a3f896e2be1827)
set(src "${CMAKE_CURRENT_BINARY_DIR}/src/main.c")
file(READ "${src}" _c)
# Also tried `<tinyfiledialogs/tinyfiledialogs.h>`
string(REPLACE [[#include "tinyfiledialogs.h"]] [[#include <tinyfiledialogs.h>]] _c "${_c}")
file(WRITE "${src}" "${_c}")

find_package(tinyfiledialogs CONFIG REQUIRED)

add_executable("${PROJECT_NAME}" "${src}")

target_link_libraries("${PROJECT_NAME}" PRIVATE tinyfiledialogs::tinyfiledialogs)
```

```sh
$ cmake -DCMAKE_BUILD_TYPE='Debug' -DCMAKE_TOOLCHAIN_FILE='[path]/vcpkg/scripts/buildsystems/vcpkg.cmake' ..
$ cmake --build .
build/src/main.c:54:10: fatal error: 'tinyfiledialogs.h' file not found
#include <tinyfiledialogs.h>
         ^~~~~~~~~~~~~~~~~~~
1 error generated.
gmake[2]: *** [CMakeFiles/pp.dir/build.make:76: CMakeFiles/pp.dir/src/main.c.o] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/pp.dir/all] Error 2
gmake: *** [Makefile:91: all] Error 2
```

**Comment by JackBoosY** [comment]
@SamuelMarks  Can you please try again?

**Comment by SamuelMarks** [comment]
@JackBoosY great that worked @ 408df6b3:
```sh
$ gh pr checkout 19287
$ ./bootstrap-vcpkg.sh
$ ./vcpkg install tinyfiledialogs
```

Then in another dir with this above it:
```cmake
cmake_minimum_required(VERSION 3.4)
cmake_policy(SET CMP0048 NEW)
project(pp VERSION 0.0.0 LANGUAGES C)

file(DOWNLOAD "https://sourceforge.net/p/tinyfiledialogs/code/ci/0002/tree/hello.c?format=raw"
              "src/main.c"
     EXPECTED_HASH SHA256=ceef2c9cb381704f9a321260a4869daa427ab4dd2b82f01e72fd768365210ccb)
set(src "${CMAKE_CURRENT_BINARY_DIR}/src/main.c")
file(READ "${src}" _c)
string(REPLACE [[#include "tinyfiledialogs.h"]] [[#include <tinyfiledialogs.h>]] _c "${_c}")
file(WRITE "${src}" "${_c}")

find_package(tinyfiledialogs CONFIG REQUIRED)

add_executable("${PROJECT_NAME}" "${src}")

target_link_libraries("${PROJECT_NAME}" PRIVATE tinyfiledialogs::tinyfiledialogs)
```

```sh
$ cmake -DCMAKE_BUILD_TYPE='Debug' -DCMAKE_TOOLCHAIN_FILE='[path]/vcpkg/scripts/buildsystems/vcpkg.cmake' ..
$ cmake --build .
[ 50%] Building C object CMakeFiles/pp.dir/src/main.c.o
[100%] Linking C executable pp
[100%] Built target pp
```

File: versions/t-/tinyfiledialogs.json [hunk 1]
```diff
@@ -1,5 +1,10 @@
 {
   "versions": [
+    {
+      "git-tree": "02f1f2a70350a2bf3701cf6279af2ab577d0e6da",
```
Comment (JackBoosY) [inline versions/t-/tinyfiledialogs.json:4]:
```suggestion
      "git-tree": "3959a47c7d93ca7db6e2022553b1d3427970cecf",
```

**Review by PhoebeHui — APPROVED** [review]

**Pull request merged**